hyrisecockpit/frontend/src/components/workload/WorkloadGeneration.vue

```diff
@@ -9,110 +9,41 @@
+        <status-warning
+          :selected-databases="databases"
+          :selected-metrics="['']"
+        />
         <v-row>
-          <v-col max-width="300px">
+          <v-col class="pt-2">
             <p class="subtitle-1 font-weight-medium">
               Start, pause and stop a workload
             </p>
-            <v-slider
-              id="frequency-slider"
-              v-model="frequency"
-              class="mt-10"
-              thumb-label="always"
-              min="0"
-              max="1000"
-              @click="handleFrequencyChange()"
-            ></v-slider>
-            <v-text-field
-              id="frequency-field"
-              v-model="frequency"
-              label="Number of queries per second"
-              outlined
-              dense
-              @change="handleFrequencyChange()"
-            ></v-text-field>
-            <v-radio-group
-              v-model="workload"
-              class="mt-0"
-              @change="handleWorkloadChange()"
-            >
-              <v-radio
-                v-for="workload in availableWorkloads"
-                :key="workload"
-                :label="getDisplayedWorkload(workload)"
-                :value="workload"
-                :disabled="
-                  !isLoaded(workload) || loadingWorkloadData || instanceBlocked
-                "
-              >
-              </v-radio>
-            </v-radio-group>
-            <v-btn-toggle>
-              <v-btn
-                id="start-workload"
-                @click="startingWorkload()"
-                :disabled="buttons.start.loading || !isLoaded(workload)"
-                :loading="buttons.start.loading"
-                :style="{ color: buttons.start.active ? 'green' : '' }"
-              >
-                <v-icon>
-                  mdi-play
-                </v-icon>
-              </v-btn>
-              <v-btn
-                id="pause-workload"
-                @click="pausingWorkload()"
-                :disabled="buttons.pause.loading || !isLoaded(workload)"
-                :loading="buttons.pause.loading"
-                :style="{ color: buttons.pause.active ? 'blue' : '' }"
-              >
-                <v-icon>
-                  mdi-pause
-                </v-icon>
-              </v-btn>
-              <v-btn
-                id="stop-workload"
-                @click="stoppingWorkload()"
-                :disabled="buttons.stop.loading || noDatabaseAdded"
-                :loading="buttons.stop.loading"
-                :style="{ color: buttons.stop.active ? 'red' : '' }"
-              >
-                <v-icon>
-                  mdi-stop
-                </v-icon>
-              </v-btn>
-            </v-btn-toggle>
+            <frequency-handler @change="handleFrequencyChange($event)" />
+            <workload-selector
+              :workload-data="workloadData"
+              :disabled="disabled"
+              @change="handleWorkloadChange($event)"
```

Comment:
```suggestion
              @change="handleWorkloadChange"
```
Thats exactly the same, but prettier :smile:

Reply:
Thanks!